@navikt/core/react/src/link/Link.tsx

```diff
@@ -4,6 +4,16 @@ import { OverridableComponent } from "../util/OverridableComponent";
 
 export interface LinkProps
   extends React.AnchorHTMLAttributes<HTMLAnchorElement> {
+  /**
+   * Whether the link is suitable for being inlined in text.
+   * @default false
+   */
+  inline?: boolean;
+  /**
+   * Whether the link should break on all characters. Useful for long links with no "words".
+   * @default false
+   */
+  "break-All"?: boolean;
```

Comment:
Ønsker vi dette som en egen prop? Som standard arver den fra parent, slik at bruker vil styre dette på paragraf/seksjon-nivå 🤔

Reply:
Tanken var at man vil kanskje bare ha `break-all` i veldig få cases tenker jeg? (lange linker uten word breaks kan gi stygg / rar layout av tekst. Så man vil ha det så begrenset som mulig (skal ikke gjelde for hele paragrafet / seksjonen). Men er det vanlig å ha slike lange lenker anyways? Ville tro at lenker burde skrives "pent" som content ellers. Så tenker jeg bare fjerner denne propen også. Da oppfordrer vi mot å skrive fine lenker med å ikke tilrettelegge for å lage stygge lenker 😂 .